core/os/file_access.h

```diff
@@ -140,7 +140,7 @@ class FileAccess {
 
 	virtual Error reopen(const String &p_path, int p_mode_flags); ///< does not change the AccessType
 
-	virtual Error _chmod(const String &p_path, int p_mod) {}
```

Comment:
Should it return `OK` or `FAILED` by default? @marcelofg55

Reply:
Should return FAILED by default. I missed that.